frontend/packages/noobaa-storage-plugin/src/components/details-card/details-card.tsx

```diff
@@ -14,23 +14,20 @@ import {
 import { FirehoseResource } from '@console/internal/components/utils';
 import { InfrastructureModel } from '@console/internal/models';
 import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
-import { CephClusterModel } from '../../../../ceph-storage-plugin/src/models';
-
-const getInfrastructurePlatform = (infrastructure: K8sResourceKind): string =>
-  _.get(infrastructure, 'status.platform');
-
-const getCephVersion = (cephCluster: K8sResourceKind): string =>
-  _.get(cephCluster, 'spec.cephVersion.image');
+import { CSVModel } from '../../../../ceph-storage-plugin/src/models';
+import { OCS_NAMESPACE } from '../../../../ceph-storage-plugin/src/constants/index';
+import {
+  getInfrastructurePlatform,
+  getOCSOperatorVersion,
+} from '../../../../ceph-storage-plugin/src/util';
```

Comment:
define dependency in `package.json` for    `"@console/ceph-storage-plugin": "0.0.0-fixed"` and then import 
```suggestion
} from '@console/ceph-storage-plugin/src/util';
```

Reply:
Use same for all.

Reply:
ack